### DISCUSSION TITLE
Deployment Issue of Adonis 6 Application on DigitalOcean and Render

### DISCUSSION THREAD

**Discussion by gabrielbuzziv** [adonisjs/core#4473] [Help]
Hello, fellow developers!

I'm currently experiencing a deployment issue with an Adonis 6 application on DigitalOcean and Render. The build and running phases seem to execute properly, but upon completion, the following message is consistently returned:

`﻿[10:50:11.479] INFO (51): started HTTP server on localhost:8080.`

Unfortunately, despite this message, both DigitalOcean and Render constructors fail to build the application successfully.

My setup is as follows:

**Build Command:** ﻿`yarn build`

**Run Command:**
```
node ace migration:run --force 
cd build 
yarn start --production
````

The above commands work perfectly in a local environment, but they seem to fail during the deployment stage on both DigitalOcean and Render.

Thats the deployment logs
![](https://i.imgur.com/QIDyLUC.png)

Has anyone experienced a similar issue or does anyone have any recommendations for troubleshooting this? Any help would be greatly appreciated!

Thank you!

**Comment by orafal-dev** [comment]
Probably you could omit `cd build` step in your run command. Since `yarn start` will run `node build/bin/server.js` (Or at least I have it setup like tihs, and it's deploying)
